.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Java CI with Maven
```

Comment:
Might want to push it to your fork and run it there, it looks like this wont run without approval here. It _might_ run if you use the same file name as the one that we merged a few days ago though 🤔

Reply:
I tried but it doesn't build the Windows version, we'll need a Windows runner and a Visual Studio dev console for that. Should work now 👌